app/models/enforcement.rb

```diff
@@ -11,6 +14,10 @@ class Enforcement < ApplicationRecord
       postcode: :postcode
     }
 
+  alias_attribute :address_1, :address_line_1
+  alias_attribute :address_2, :address_line_2
+  alias_attribute :description, :breach_description
```

Comment:
I'd like to avoid introducing compatibility aliases if we can help it, but maybe it's worth it temporarily to get this done quickly?

(If we migrate the PlanningApplication to use the address model, then we can refer to `address.line_1` etc on both PlanningApplication and Enforcement, perhaps.)

Reply:
Happy to do that as a separate PR and then revert/refactor this change